_posts/2015-10-21-Onefootbot.md

```diff
@@ -0,0 +1,123 @@
+      'It is better to win ten times 1-0 than to win once 10-0. - Vahid “Vaha” Halilhodžic',
+      'Football is a simple game; 22 men chase a ball for 90 minutes and at the end, the Germans win. - Gary Lineker',
+      'In Europe, it’s different – you eat soccer, you breathe soccer, you drink soccer. Everything is about soccer. - Thomas Dooley',
+      'We must have had 99 percent of the game. It was the other three percent that cost us the match. - Ruud Gullit',
+      'Sometimes in soccer you have to score goals.- Thierry Henry',
+      'At the World Cup I watched the game Austria against Cameroon. On the one side exotics, alien culture , wild rites - and on the other side : Cameroon ! - Dieter Nuhr',
+      'The subjunctive is the enemy of the losers. - Jens Lehmann'
+    ]
+
+    num = randomNum(quotes.length)
+
+    quotes[num]
+
+  randomNum = (max,min=0) ->
+    Math.floor(Math.random() * (max - min) + min)
+```
+
+Just imagine how much you can annoy your co-workers by doing this in a football based company :-). Basically we had to
+change the behaviour so that the robot only responds with football quotes when asked directly ;-).
+
+### A more serious use case
+
+So far that was a lot of fun but of course we also had a more serious use case in mind when introducing Onefootbot. The
+goal was to improve our on call process so that all people in our company are able to find out who is currently on call 
+in case of an incident and additionally be able to track the status of an incident so he/she does not have to contact the 
+engineer who is on call every time an update is needed.
+
+We are managing our on call schedule via [pagerduty](https://www.pagerduty.com/) so we used the existing 
+[pagerduty integration for Hubot](https://github.com/hubot-scripts/hubot-pager-me) in order to connect Onefootbot with 
+pagerduty. Together with the [HipChat integration of pagerduty](https://www.pagerduty.com/docs/guides/hipchat-integration-guide/) 
+we were able to achieve what we wanted. All employees of Onefootball are able to discover who is currently on call and to 
+follow the status of an incident just by using the internal company chat. This is especially useful on weekends. Even so 
+you might not have your laptop with you all the time you can simply use the HipChat client on your mobile phone in order
+to check for the things mentioned above.
+
+All that helped a lot to make our on call process and incident management more transparent to all members of our company. 
+But we noticed pretty soon that we could also use the bot to achieve some improvements for the persons being on call 
+themselves. We were missing some functionality in the pagerduty script for Hubot in order to check for the on call members 
+if they are on call and how does their on call schedule look like. So we contributed the functioality to the mentioned 
+pagerduty integration script by adding these [2 commands](https://github.com/hubot-scripts/hubot-pager-me/pull/48).
```

Comment:
I'd prefer seeing commented code blocks and less text here, while the pr link is fine to have in.